strr-platform-web/app/stores/platform.ts

```diff
@@ -4,17 +4,30 @@ export const useStrrPlatformStore = defineStore('strr/platform', () => {
   const { platformBusiness } = storeToRefs(useStrrPlatformBusiness())
   const { platformDetails } = storeToRefs(useStrrPlatformDetails())
 
-  const activePlatform = ref<PlatformRegistrationPayload | ApiBaseRegistration | undefined>(undefined)
+  const activeApplicationInfo = ref<ApplicationHeader | undefined>(undefined)
+  const activePlatform = ref<PlatformRegistrationResp | ApiBaseRegistration | undefined>(undefined)
+
+  // @ts-expect-error - registration_number will only be there for registrations
+  const isRegistration = computed(() => !!activePlatform.value?.registration_number)
+
+  const isApplicationStatus = (statuses: ApplicationStatus[]) =>
+    statuses.includes(activeApplicationInfo.value?.status as ApplicationStatus)
+
+  const isPaidApplication = computed(() =>
+    activeApplicationInfo.value && !isApplicationStatus([ApplicationStatus.DRAFT, ApplicationStatus.PAYMENT_DUE]))
+
+  const showPlatformDetails = computed(() => isRegistration.value ||
+    (activeApplicationInfo.value && !isApplicationStatus([ApplicationStatus.DECLINED, ApplicationStatus.DRAFT])))
 
   const getAccountRegistrations = async () => {
     // TODO: add error handling / modal popup?
-    const resp = await $strrApi('/registrations', { method: 'GET' }) as { registrations: PlatformRegistrationPayload[] }
+    const resp = await $strrApi('/registrations', { method: 'GET' }) as { registrations: PlatformRegistrationResp[] }
     return resp?.registrations
   }
 
   const getAccountApplications = async () => {
     // TODO: add error handling / modal popup?
-    const resp = await $strrApi('/applications', { method: 'GET' }) as { applications: PlatformApplicationPayload[] }
+    const resp = await $strrApi('/applications', { method: 'GET' }) as { applications: PlatformApplicationResp[] }
```

Comment:
should be able to do this the same as $fetch -> $strrApi<{ applications: PlatformApplicationResp[] }>('/applications..

GET is also on by default